src/pages/food/containers/FoodDetailsContainer.tsx

```diff
@@ -22,19 +23,20 @@ class FoodDetailsContainer extends React.Component<Props> {
   }
 
   public render(): JSX.Element {
-    const { navigation, foodDetails } = this.props;
+    const { foodDetails, nutrition } = this.props;
 
     return (
       <FoodDetailsPage
-        onAdd={ () => navigation.navigate('StockDetails', { foodId: foodDetails.id }) }
         foodDetails={ foodDetails }
+        nutrition={ nutrition }
```

Comment:
shouldn't the nutrition already in the foodDetails?

Reply:
I'm a bit confused by this. I thought nutrition wasn't ready yet, so I thought this was fine for just the mock data

Reply:
Yes, this way is fine as well. I just didn't want unused props lying around. But you are right